modules/cms/classes/Theme.php

```diff
@@ -181,7 +181,7 @@ public static function getActiveThemeCode()
          * If a value is returned from this halting event, it will be used as the active
          * theme code. Example usage:
          *
-         *     Event::listen('cms.theme.getActiveTheme', function() { return 'mytheme'; });
+         *     Event::listen('cms.theme.getActiveTheme', function () { return 'mytheme'; });
```

Comment:
@DanHarrin Could you add new lines between the braces so that it is consistent with the other examples?
```suggestion
         *     Event::listen('cms.theme.getActiveTheme', function () {
         *         return 'mytheme';
         *     });
```